soh/soh/Enhancements/randomizer/3drando/hint_list.cpp

```diff
@@ -28,12 +28,12 @@ void HintTable_Init() {
     hintTable[PREFIX] =
         HintText::Exclude({ Text{ "They say that ", /*french*/ "Selon moi, ", /*spanish*/ "Según dicen, " } });
     hintTable[WAY_OF_THE_HERO] =
-        HintText::Exclude({ Text{ " is on the way of the hero.", /*french*/ " est sur la voie du héros.",
+        HintText::Exclude({ Text{ " is on %cthe way of the hero%w.", /*french*/ " est sur la voie du héros.",
                                   /*spanish*/ " conduce a la senda del héroe." } });
     hintTable[PLUNDERING] =
         HintText::Exclude({ Text{ "plundering ", /*french*/ "explorer ", /*spanish*/ "inspeccionar " } });
     hintTable[FOOLISH] = HintText::Exclude(
-        { Text{ " is a foolish choice.", /*french*/ " est futile.", /*spanish*/ " no es una sabia decisión." } });
+        { Text{ " is %pa foolish choice%w.", /*french*/ " est futile.", /*spanish*/ " no es una sabia decisión." } });
```

Comment:
est %pfutile%w.

Reply:
Missed 2 of them there actually, good call. 👍